snuba/migrations/runner.py

```diff
@@ -47,6 +53,34 @@ def __init__(self) -> None:
             ClickhouseClientSettings.MIGRATE
         )
 
+    def show_all(self) -> List[Tuple[MigrationGroup, List[MigrationStatus]]]:
+        """
```

Comment:
Can that List be a Sequence instead ?

Reply:
mypy complains about using `.append()` on Sequence